src/services/read-from-disk.service.js

```diff
@@ -206,14 +211,24 @@ export function loadAllProjectDependencies(
   return loadPackageJson(projectPath).then(
     packageJson =>
       new Promise((resolve, reject) => {
-        const dependencyNames = Object.keys(packageJson.dependencies);
+        // Check for existence of both dependencies and devDependencies
+        // We can reasonably assume all projects have dependencies
+        // but some may not have devDependencies
+        const deps = Object.keys(packageJson.dependencies);
+        const devDeps = packageJson.devDependencies
+          ? Object.keys(packageJson.devDependencies)
+          : [];
```

Comment:
Nit: this is totally a matter of style, but I think I have a slight preference for:

```js
const devDeps = Object.keys(packageJson.devDependencies || []);
```

I like it because it's more concise. It's true that it does mean the computer will be doing a bit more work, but it's completely negligible at this scale.

Reply:
So essentially that reads

Reply:
Nice. Forgot about that. Looks much cleaner, I agree.